client/src/main/java/io/pravega/client/admin/StreamManager.java

```diff
@@ -249,4 +249,14 @@ static StreamManager create(ClientConfig clientConfig) {
      */
     @Override
     void close();
+
+    /**
+     * Fetch the distance between two streamCuts.
+     *
+     * @param stream The stream with streamCuts.
```

Comment:
Does stream has streamCuts?

Reply:
streamcut as an object is related to streams. But streams don't directly deal with streamcuts. It is more related to segments.
@tkaitchuck please correct if wrong, or if anything more needs to be added.

Reply:
Perhaps "The stream which the streamCuts come from" or "The stream corresponding with the streamCuts"